test/integration/nsmd_test_utils/test_utils.go

```diff
@@ -464,14 +464,10 @@ func checkNSCConfig(k8s *kube_testing.K8s, t *testing.T, nscPodNode *v1.Pod, che
 
 func checkVppAgentNSCConfig(k8s *kube_testing.K8s, t *testing.T, nscPodNode *v1.Pod, checkIP string) *NSCCheckInfo {
 	info := &NSCCheckInfo{}
-	for attempts := 0; attempts < 30; attempts++ {
-		response, errOut, _ := k8s.Exec(nscPodNode, nscPodNode.Spec.Containers[0].Name, "vppctl", "show int addr")
-		if strings.Contains(response, checkIP) {
-			info.ipResponse = response
-			info.errOut = errOut
-			break
-		}
-		time.Sleep(time.Millisecond * 100)
+	response, errOut, _ := k8s.Exec(nscPodNode, nscPodNode.Spec.Containers[0].Name, "vppctl", "show int addr")
```

Comment:
Hmm can you elaborate why you reduce the attempts to one? Also see #887 where the attempts are increased to 60.

Reply:
Because before we didn't wait for memif interface setup. The attempts were needed only for memif interface setup.
I've added waiting in https://github.com/networkservicemesh/networkservicemesh/pull/925.